static/js/zamboni/files.js

```diff
@@ -112,6 +112,34 @@ var Highlighter = {
         // highlighter will try to replace the original element in the
         // DOM when it's done.
 
+        // Verify that we actually support the used brush
+        // see https://github.com/mozilla/addons-server/issues/4552 for more
+        // details. This shows an alert if a brush is not supported.
+        var discoveredBrushes = SyntaxHighlighter.brushes;
+
+        if (discoveredBrushes) {
+            var brushes = {};
+
+            for (var discoveredBrush in discoveredBrushes) {
+                var aliases = discoveredBrush.aliases;
+
+                if (aliases == null) {
+                    continue;
+                }
+
+                for (var i = 0, l = aliases.length; i < l; i++) {
+                    brushes[aliases[i]] = discoveredBrush;
+                }
+            }
+
+            if (discoveredBrushes[brushes[brush]] == null) {
+                window.alert(format(
+                    'Could not find brush for: {0}. The output can be broken, please be careful!',
```

Comment:
I like your suggestion, I'll maybe go with ’File content not supported. The output can be broken, please be careful and report an issue immediately!’ - since, if that alert ever happens someone added or removed something from the python mapping without taking care of the javascript-side of things so the "report an issue" part is kinda important imho?